lib/person.rb

```diff
@@ -0,0 +1,45 @@
+require 'date'
+
+class Person
+
+  attr_accessor :available_books, :shelf, :library_account
+  def initialize
+    @shelf = YAML.load_file('./books/data.yml')
+    @available_books = @shelf.select { |obj| obj[:available]}
+  end
+
+  def available
+    if @available_books
+      'Book is available for borrowing'
+    else
+      'Book is not available for borrowing'
+    end
+  end
```

Comment:
?? Isn't this a method that should be in the Library class? I can't see how this can be a responsibility of a Person?